SetGridField fixes (#31318)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_12_56.md

#### Scripts

##### SetGridField

> - Fix an issue that the script raised an uninformative error when trying to run SetGridField from the playground and from the debugger.

Reviewer: ```suggestion
- Fixed an issue that the script raised an uninformative error when trying to run the script from the playground and from the playbook debugger.
```

> - Fixed an issue that the script raised an uninformative error when trying to run the script from the playground and from the playbook debugger.

Reviewer 2: ```suggestion
- Fixed an issue where an uninformative error was raised when trying to run the script from the playground or the playbook debugger.
```
